tools/sof-tplgreader.py

```diff
@@ -202,12 +206,12 @@ def func_getPipeline(tplgObj, tplgName, sdcard_id, sort):
     parser.add_argument('filename', type=str, help='tplg file name, multi-tplg file name use "," to split it')
     parser.add_argument('-s', '--sofcard', type=int, help='sofcard id', default=0)
     parser.add_argument('-f', '--filter', type=str, nargs='+',
-        help='''setup filter, this value is format value,
+        help='''setup filter, command line parameter
 string format is 'key':'value','value'
```

Comment:
I think you need to add 'any' as 'special value'

Reply:
I will add some text to explain 'any' in help info